__tests__/api/AnalysesApi.test.ts

```diff
@@ -0,0 +1,38 @@
+import AnalysesApi from '@/api/AnalysesApi'
+import { AnalysisStrategy, RecommendationReason, RecommendationWarning } from '@/models'
+
+describe('AnalysesApi.ts module', () => {
+  describe('findByExperimentId', () => {
+    it('should return a set of analyses with the expected shape', async () => {
+      const analyses = await AnalysesApi.findByExperimentId(123)
+      expect(analyses).toBeDefined()
+      expect(Array.isArray(analyses)).toBe(true)
+      expect(analyses.length).toBeGreaterThan(0)
+      analyses.forEach((analysis) => {
+        expect(typeof analysis.metricAssignmentId).toBe('number')
+        expect(Object.values(AnalysisStrategy).includes(analysis.analysisStrategy)).toBe(true)
+        expect(typeof analysis.participantStats['total']).toBe('number')
+        expect(typeof analysis.participantStats['not_final']).toBe('number')
```

Comment:
These don't require bracket notation. There is an ESLint we can add to require/fix not use the more verbose bracket notation when not needed. (I seems like you are switching language contexts. Not Python here but JavaScript :-))
```suggestion
        expect(typeof analysis.participantStats.total).toBe('number')
        expect(typeof analysis.participantStats.not_final).toBe('number')
```

Reply:
Yeah, I used the quoted version because `participantStats` is more like an open-ended mapping. This makes it more explicit, but I don't mind using the dot notation if you like it better. :slightly_smiling_face: